sdk/rust/src/contract/contract_function_parameters.rs

```diff
@@ -0,0 +1,1268 @@
+    pub fn add_uint200_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint200[]", 25)
+    }
+
+    /// Add a `uint208[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint208_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint208[]", 26)
+    }
+
+    /// Add a `uint216[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint216_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint216[]", 27)
+    }
+
+    /// Add a `uint224[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint224_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint224[]", 28)
+    }
+
+    /// Add a `uint232[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint232_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint232[]", 29)
+    }
+
+    /// Add a `uint240[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint240_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint240[]", 30)
+    }
+
+    /// Add a `uint248[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint248_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint248[]", 31)
+    }
+
+    /// Add a `uint256[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint256_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint256[]", 32)
+    }
+
+    /// Add an `address` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_address(&mut self, address_string: &str) -> &mut Self {
+        self.args.push(Argument {
+            type_name: "address",
+            value_bytes: Self::encode_address(address_string),
+            is_dynamic: false,
+        });
+        self
+    }
+
+    /// Add an `address[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_address_array(&mut self, address_strings: &[&str]) -> &mut Self {
+        self.args.push(Argument {
+            type_name: "address[]",
+            value_bytes: Self::encode_array_of_32_byte_elements(
+                address_strings.iter().map(|addr| Self::encode_address(addr)),
+                address_strings.len(),
+            ),
+            is_dynamic: true,
+        });
+        self
+    }
+
+    #[allow(dead_code)]
+    pub fn add_function(
+        &mut self,
+        address_string: &str,
+        mut selector: ContractFunctionSelector,
+    ) -> &mut Self {
+        let mut value_bytes = IdEvmAddress::from_str(address_string).unwrap().0 .0.to_vec();
```

Comment:
```suggestion
        let mut value_bytes = IdEvmAddress::from_str(address_string).unwrap().to_bytes();
```
you don't use anything vec specific anywhere